github/Services/GithubServiceDecorator.php

```diff
@@ -141,32 +141,36 @@ private function pullRequestDecorator(CursorPaginator $result): CursorPaginator
             /** @var OctoMember $member */
             $member = $members->get($pullRequest?->author?->login);
 
-            $author = Member::fromOctoMember(
-                $pullRequest->author,
-                $pullRequest?->author?->login ? $member : null
-            );
+            if (!is_null($member)) {
+                $author = Member::fromOctoMember(
+                    $pullRequest->author,
+                    $member
+                );
+            } else {
+                $author = Member::fromOctoMember(
+                    $pullRequest->author,
+                    $member
+                );
+            }
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**🔨 issue:** ‏C'est le même code dans le if et le else